ocaml/typing/typedecl.ml

```diff
@@ -2328,14 +2416,22 @@ let transl_value_decl env loc valdecl =
         | Native_repr_attr_present repr -> Some repr
         | Native_repr_attr_absent -> None
       in
+      let is_layout_poly =
+        Attr_helper.has_no_payload_attribute
```

Comment:
I think using `Builtin_attributes.has_attribute` here would be slightly more efficient.  I am actually wondering what purpose `Attr_helper` is serving at this point, but that's a question for a different day.

Reply:
Changed it to use `Builtin_attributes.has_attribute`. Added the attribute to the `builtin_attrs` table. And wrote w53 tests.